static/cheatcodes/.htaccess

```diff
@@ -0,0 +1,2 @@
+RewriteEngine on
+RewriteRule ^$	https://git.grml.org/f/grml-live/templates/GRML/grml-cheatcodes.txt
```

Comment:
This needs to be `https://git.grml.org/f/grml-live/config/media-files/GRMLBASE/GRML/GRML_NAME/grml-cheatcodes.txt` instead

(And while at it, please clarify the details in the commit message, to provide some context and further information around it!)